lib/config/presets/common.ts

```diff
@@ -23,3 +23,39 @@ export const removedPresets = {
   'helpers:oddIsUnstable': null,
   'helpers:oddIsUnstablePackages': null,
 };
+
+const renamedMonorepos: Record<string, string> = {
+  'arcus event-grid': 'arcus.event-grid',
+  'arcus security': 'arcus.security',
+  'arcus messaging': 'arcus.messaging',
+  'arcus observability': 'arcus.observability',
+  'arcus webap': 'arcus.webap',
```

Comment:
```suggestion
  'arcus webapi': 'arcus.webapi',
```

I think a typo slipped in, my original PR https://github.com/renovatebot/renovate/pull/10243/files made the change from `arcus webapi` -> `arcus.webapi`